Empty Payload Yields 500

```
AssertionError: undefined == true
    at /usr/src/app/server/api/tunnel.js:12:5
    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/src/app/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/usr/src/app/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)
    at /usr/src/app/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/usr/src/app/node_modules/express/lib/router/index.js:330:12)
    at next (/usr/src/app/node_modules/express/lib/router/index.js:271:10)
    at Function.handle (/usr/src/app/node_modules/express/lib/router/index.js:176:3)
    at router (/usr/src/app/node_modules/express/lib/router/index.js:46:12)
```

--- a/server/api/tunnel.js
+++ b/server/api/tunnel.js
@@ -1,4 +1,3 @@
-import assert from 'assert';
 import { Router } from 'express';
 
 import { getNetwork } from '../lib/network-registry';
@@ -9,7 +8,9 @@ export default function () {
 
   api.put('/', (req, res) => {
     const { hostname, port } = req.body;
-    assert(hostname, port);
+    if (!hostname || !port) {
+      return res.status(400).send('invalid hostname or port');
+    }
 
     const network = getNetwork(req.networkId);
     if (!network) {

